grails-app/i18n/messages.properties

```diff
@@ -1122,6 +1122,7 @@ cycleCountStartRecountCommand.cycleCountRequest.noCycleCountFound=Cannot start r
 cycleCountStartRecountCommand.cycleCountRequest.invalidCycleCountStatus=Recount cannot be started when cycle count status is ({3})
 cycleCountStartRecountCommand.countIndex.invalid=Count index ({3}) is invalid for recount
 cycleCountSubmitCountCommand.cycleCount.invalidStatus=Invalid cycle count status to be able to submit the count
+cycleCountSubmitRecountCommand.cycleCount.invalidStatus=Invalid cycle count status to be able to submit the recount
```

Comment:
We've never adopted a standard naming convention for command classes before (and we can discuss this convention later). Throughout the application, we go back and forth between resource vs action so it doesn't really matter at this point. But this seems more like a combination of the action convention (SubmitRecount) with a namespace (CycleCount).

I like the action-oriented command class naming convention. 

So I think one of the following would 
* SubmitCycleCountRecountCommand 
* SubmitRecountCommand 

But would be fine with CycleCountRecountCommand as well.